advisories/a97090.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Technical Advisory 97090
+advisory: A-97090
+summary: Queries planned with a zigzag join could produce incorrect results if the two indexes used for the join had a matching suffix of index key columns where the direction was different between the two indexes.
+toc: true
+affected_versions: v19.1 to v22.1.15, v22.2.0 to v22.2.5
+advisory_date: 2023-02-24
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+In all versions of CockroachDB from [v19.1](../releases/index.html#v19-1) to [v22.1.15](../releases/v22.1.html#v22-1-15) and [v22.2.0](../releases/v22.2.html#v22-2-0) to [v22.2.5](../releases/v22.2.html#v22-2-5), queries that used a zigzag [join](../{{site.versions["stable"]}}/joins.html) could return incorrect results in some cases. Zigzag joins can be planned when at least two columns from the same table are fixed to constant values by the query predicate, and indexes exist on both of those columns. The bug could occur when the two indexes used had a matching suffix of key columns where the direction of at least one column differed between the two indexes. 
```

Comment:
first mention of "zigzag join" could use the more specific link to zigzag join info at https://www.cockroachlabs.com/docs/stable/cost-based-optimizer.html#zigzag-joins instead of the less specific "joins" page

(Aside: I noticed while searching for the link above that the joins page list of join algorithms does not mention zigzag joins in the listed join types so I filed https://cockroachlabs.atlassian.net/browse/DOC-7063 - no need to do anything in this PR but just sharing info FYI)